_data/CSElonewolf/CSElonewolf.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Devkant Gorai 
+institution: Government College of Engineering and Textile Technology
+profile_pic: CSElonewolf.jpg
```

Comment:
Incorrect file name, change `CSElonewolf.jpg` to `CSELonewolf.jpg` or rename the image to `CSElonewolf.jpg`